### PR TITLE
Readme hk

### LINKED ISSUE #101: ReadMe: Wins & Challenges
- Multiple groups working with 1 api: API Data -- ingredients in pantry with 0 after adding/removing pantry items
because people in different groups are modifying data in different ways
- test data and API data for users array not matching -- different properties in each ingredient, multiples of each ingredient object (RIP Marie Condo methods)
- writing our tests based the wrong data -- we should've used test data based on samples of the API instead of reusing the data that came with the repo
- figuring out fetch API flow
- learning how to taking advantage of webpack features - not having to link everything individually / figuring out what needs to be linked with entrypoint / html for webpack to bundle app properly for deployment
- deciding (not) to use inheritance
- double datasets in Pantry class

### LINKED ISSUE #102: ReadMe: additional links section
https://jamboard.google.com/d/1k-Fsp7zcx3fZqDgq3sIR5eu4ZKydwCJyzZFebz4ChQg/viewer?f=3
https://github.com/npdarrington/Whats-Cookin/projects/1
https://frontend.turing.io/projects/module-2/refactor-tractor-wc.html
https://frontend.turing.io/projects/whats-cookin.html

### LINKED ISSUE #103: ReadMe: Technologies used
- Vanilla JS
- HTML 
- CSS/SASS
- Webpack
- TDD: Mocha / Chai / Spies
- NPM
- Fetch API - GET/POST data to a server

Anything missing?

### LINKED ISSUE #104: ReadMe: Skills used
- ES6 Classes
- array iterator methods/object methods
- TDD - test first, pseudocode, write function
- ARIA / accessibility - colorblindness, tabbing for screen readers
- group workflow - concise commits, linking commits/PR's to issues, auto PR template, 
- refactoring

### PR DESCRIPTION
**Pull request by hannakim91**
### What’s this PR do?  
write up readme for the project
### Where should the reviewer start?  
- readme.md
### How should this be manually tested?  
open up readme file, or MD preview 
### Any background context you want to provide?  
this **is** the background context
### What are the relevant tickets?  
Resolves #101 
Resolves #102 
Resolves #103 
Resolves #104
### Questions: 
- check for legibility/typos
- still need to finish abstract
- need gif/screenshots for working app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,56 @@
-# Webpack Starter Kit
+# Can You Smell What's Cookin'?
```
Comment (JWoltjen) [inline README.md:1]: Nice title name!

File: README.md [hunk 2]
```diff
@@ -1,105 +1,56 @@
...
-After one person has gone through the steps of cloning down this repo and editing the remote, everyone should clone down the repo. 
-
-Then install the library dependencies. Run:
-
-```bash
-npm install
+Refactor existing codebase for the app using technologies such as API Fetch, SASS, and Webpack...
+built with JS... 
+### App in Action
+//screenshots and/or gif's here
```
Comment (JWoltjen) [inline README.md:9]: I can help implement the gifs if y'all need help with that.

File: README.md [hunk 3]
```diff
@@ -1,105 +1,56 @@
...
+- Fetch API to GET/POST data to a server
+- GitHub/Git
+- Eslint
+## Wins & Challenges
+### Challenges
+- Writing our tests based the wrong data -- in hindsight, we should have checked the API data supplied before writing new tests and methods. We learned later in the process that API datasets differed from original project's repo data and could have saved time with functionality that was unneeded (RIP `User.marioCondoMyPantry`)
+- Using API data that can be modified by multiple developers (both within our group and other students in the class). We're all learning how to use API data for the first time, so data was sometimes altered in ways that might not work with how our class methods were written.
+### Wins
+- Tackling complicated logic involving multiple datasets.
+- Learning how to taking advantage of Webpack features to consolidate our files into one neat bundle. We found that while everything doesn't have to be linked through a `html scripts` tag, the correct files must be linked in `index.js` and a chosen entry point (`scripts.js`) before deployment.
```
Comment (JWoltjen) [inline README.md:44]: This section is really great. Encapsulates the struggle well.

**Review by JWoltjen — APPROVED** [review]
Thanks for taking point on the readme! Let me know if you guys want any help with the gifs!

**Review by npdarrington — APPROVED** [review]
Fixed commit merge.

**Review by aemcdonald — APPROVED** [review]
Looks great! I will work out our abstract & make a few edits to help polish it up.

**Pull request merged**